hw/drivers/flash/spiflash/src/spiflash.c

```diff
@@ -644,11 +662,15 @@ spiflash_power_down(struct spiflash_dev *dev)
 
     spiflash_lock(dev);
 
+#if MYNEWT_VAL(BUS_DRIVER_PRESENT)
+    bus_node_simple_write(&dev->dev.bnode.odev, cmd, sizeof(cmd));
```

Comment:
use `(struct os_dev *)&dev->dev` - code outside bus driver should not access contents of bus node object and they are defined in a way that such cast is always possible by design

Reply:
done